p2p/server_nat_test.go

```diff
@@ -75,6 +75,65 @@ func TestServerPortMapping(t *testing.T) {
 	}
 }
 
+func TestExtipLoop(t *testing.T) {
```

Comment:
I approved, but I don't think we need this test. It looks kind of fragile.

Reply:
I was thinking the same, on the other hand for such a crucial think I wanted to be sure that it won't happen again. 

I agree that this test is not clear what it tests and that the code logic might change in ways that this test will break. I am fine to remove, as I couldn't come with a clearer way to test this path. 

Give me a green light and I will remove it.

Reply:
Fixed by [b2eb0c8](https://github.com/ethereum/go-ethereum/pull/28911/commits/b2eb0c848f5b1f9f03164276a3d33a83aea95703)